Unix/base/messages.h

```diff
@@ -195,6 +195,14 @@ struct _Message
      */
     MI_Char *shellId;
 
+    /*
+        For http transport operations that go through a load balancer,
+        we need the MS_WSMAN cookie for subsequent requests.
+        The value is retrieved from the HttpHeaders passed to
+        _HttpProcessRequest.
+    */
+    MI_Char *sessionCookie;
+
```

Comment:
Could you move this new member to end of structure?  We've seen this before, where adding an item in middle of struct causes backward incompatibility issues.

Reply:
@palladia: I made the change but I don't see how it solves anything. Message is referenced inline by all other messages so there will be size and alignment changes regardless.  Is there some other backwards compatibility issue you're thinking about?